package.json

```diff
@@ -17,7 +17,6 @@
     "test"
   ],
   "dependencies": {
-    "babel-core": "^6.4.5",
```

Comment:
This dep is also in the devDeps, so I removed it from here. I am assuming that this package is just for Development purposes. If not, I can move it into deps and remove it from devDeps.